app/src/components/inputs/inputWithIcon/inputWithIcon.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Input } from 'components/inputs/input';
+import styles from './inputWithIcon.scss';
+
+const cx = classNames.bind(styles);
+
+export const InputWithIcon = ({ icon, iconClass, ...rest }) => (
+  <React.Fragment>
+    <Input {...rest} />
+    <span className={cx('icon', iconClass)}>{icon}</span>
```

Comment:
what is iconClass for?

Reply:
Needs to specify icon styles (in my case it specify font properties for icon)

Reply:
InputWithIcon responsible to draw got with props, already styled in icon inside.

<InputWithIcon icon={<i className={cx('percent-icon')} >%</i>} >
and define styles for percent-icon component which using InputWithIcon

Reply:
propTypes = {
 icon: PropTypes.element,
}

Reply:
Thank you. I will thinking about it next time